000_example/español/AnaColon/resumen de Lesser MP

```diff
@@ -0,0 +1,25 @@
+El blanqueamiento de arrecifes de coral es la explosión masiva de dinoflagelados o el perdimiento de los pigmentos fotosintéticos en las zooxantelas. Blanqueamiento ocurre cuando la temperatura sube (lo más mínimo de 1-2 °C) de la temperatura media más alta, radiación UV, cambios en la salinidad, sedimentación pesada, y exposición a contaminantes. 
```

Comment:
```suggestion
El blanqueamiento de arrecifes de coral es la expulsión masiva de dinoflagelados o el perdimiento de los pigmentos fotosintéticos en las zooxantelas. Blanqueamiento ocurre cuando la temperatura sube (lo más mínimo de 1-2 °C) de la temperatura media más alta, radiación UV, cambios en la salinidad, sedimentación pesada, y exposición a contaminantes. 
```

Reply:
i think by using 'dinoflagelados' and 'zooxantelas' separately here, it might imply to a novice that these are separate things. Maybe use the same word or a pronoun or something to make it clear that the expulsion and the loss of pigment occur to the same thing. A separate sentence could relate the two terms or explain why they both exist and how they're slightly different

Reply:
I think expulsión is correct.